igarss_2023/igarss_mtedgecumbe_ts_analysis.ipynb

```diff
@@ -451,7 +452,7 @@
    "id": "81fcfe7b-951e-448e-8fa0-1f807712d9b7",
    "metadata": {},
    "source": [
-    "The first thing to check is that you have a fully connected network (e.g., every date in your analysis window is covered by at least one interferogram). This can be a challenge to ensure sometimes due to seasonal decorrelation issues. In snowy areas like Mt. Edgecumbe, the presence of snow on the ground introduces significant decorrelation effects that hinder the accurate measurement of surface displacements. As a result, interferograms generated during winter periods tend to be unreliable. To overcome this challenge, it becomes necessary to exclude interferograms from the winter season. However, excluding winter interferograms leaves a data gap that can hamper your connectivity. To address this issue, you can create longer baseline interferograms, which span a longer period of time and cover the missing winter period. By including a few inteferograms with long temporal baselines, you can ensure connectivity while also not including any highly decorrelated interferograms. See the graph below to see the network that Grapenthin et al. used."
+    "The first thing to check is that you have a fully connected network (e.g., every date in your analysis window is covered by at least one interferogram). This can sometimes be a challenge, due to decorrelation caused by seasonal differences. In snowy areas like Mt. Edgecumbe, the presence and/or condition of snow on the ground introduces significant decorrelation effects that hinder the accurate measurement of surface displacements. As a result, interferograms generated during winter periods tend to be unreliable. To overcome this challenge, it becomes necessary to exclude interferograms from the winter season. However, excluding winter interferograms leaves a data gap that can hamper your connectivity. To address this issue, you can create longer baseline interferograms, which span a longer period of time and cover the missing winter period. By including a few interferograms with long temporal baselines, you can ensure connectivity while also not including any highly decorrelated interferograms. See the graph below to see the network that Grapenthin et al. used."
```

Comment:
@forrestfwilliams Not sure why the changes aren't highlighted for this entry. I think the only changes are to the second and third sentences.